lib/workos/hackney_client.ex

```diff
@@ -0,0 +1,34 @@
+defmodule WorkOS.HackneyClient do
+  @behaviour WorkOS.HttpClient
```

Comment:
Reference regarding how behaviors work in Elixir: https://elixirschool.com/en/lessons/advanced/behaviours